src/main/scala/package.scala

```diff
@@ -4,13 +4,26 @@ import ohnosequences.awstools.s3._
 
 package object rna16s {
 
-  val dbName = "ohnosequences.db.rna16s"
+  type +[A, B] = Either[A, B]
 
-  private val metadata = generated.metadata.rna16s
+  implicit class PredicateOps[X](val p: X => Boolean) {
+
+    def &&(other: X => Boolean): X => Boolean =
+      x => p(x) && other(x)
+  }
+
+  val dbName: String =
+    "ohnosequences.db.rna16s"
```

Comment:
Why is this string needed?

Reply:
it's not; but it will. Let's drop it in the meantime.

Reply:
done.